_rules/audio-as-media-alternative-afb423.md

```diff
@@ -4,7 +4,9 @@ name: Audio-only as a media alternative for text
 rule_type: atomic
 description: |
   This rule checks `audio` is a media alternative for text on the page.
-test_aspects:
+accessibility_requirements:
+  
```

Comment:
@ShadowBB, should this rule have a technique as it's accessibility requirement?

Reply:
> 
> 
> @ShadowBB, should this rule have a technique as it's accessibility requirement?

No. There is no relevant technique that would necessarily fail if this rule fails.